### PR TITLE
Move fixed ccs to fixed shares

### PR DESCRIPTION
**Pull request by JakobBD**
## Purpose of this PR

A large chunk of code in the core is concerned with exogenous CCS shares for industry. 
The values coming out of it are overwritten (and therefore obsolete and unused) in the subsectors realization. 
This PR moves part of them to fixed_shares to declutter the core. Some other parts stay in core: It seems reasonable to keep the calculation of `pm_emineg_econometric` in one place. 

This branch has #1673 merged, so it should be merged after that. 

I haven't run tests of this using the fixed_shares realization, only using subsectors! 

This is part of a compromise: 
fixed_shares currently [does not compile any more](https://github.com/remindmodel/development_issues/issues/280), since I broke it with the process-based steel implementation. According to @0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q, even with a fix to that, fixed_shares would take a while to get running again, since it would need new input data, and new calibrations. subsectors being as stable as it is right now, we could think about getting rid of fixed_shares altogether sooner or later. At the same time, subsectors is quite stable right now.

The current compromise is to keep it as close to a functional state as possible without much additional work. 

My long-term proposal is to build a version of process-based industry where the process route shares are fixed, and to get rid of fixed_shares at the latest by then, maybe earlier. 

Adding @LaviniaBaumstark and @robertpietzcker to the discussion about the future of fixed_shares. 

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [ ] Bug fix 
- [x] Refactoring
- [ ] New feature 
- [ ] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [x] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [x] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here:

`/p/tmp/jakobdu/remind_temp3/output`
 
* Comparison of results (what changes by this PR?): 

no changes

### CODE REVIEW

**Comment by JakobBD** [comment]
Test runs are fine, can be merged from my point of view. 

Will become obsolete soon with deleted fixed_shares, but that deletion will take a while if done thoroughly, checking for obsolete code in core and R packages, so maybe we can merge this in the meantime.

File: modules/37_industry/subsectors/equations.gms [hunk 1]
```diff
@@ -81,23 +81,28 @@ $endif.exogDem_scen
 *' energy mix, as that is what can be captured); vm_emiIndBase itself is not used for emission
 *' accounting, just as a CCS baseline.
 ***------------------------------------------------------
-q37_emiIndBase(t,regi,entyFe,secInd37) ..
```
Comment (LaviniaBaumstark) [inline modules/37_industry/subsectors/equations.gms:84]: does it mean that the set `entyFe`can bedeleted?

**Review by LaviniaBaumstark — APPROVED** [review]

File: core/datainput.gms [hunk 2]
```diff
@@ -1359,19 +1358,18 @@ $if %cm_MAgPIE_coupling% == "off"  pm_macSwitch("co2luc") = 0;
...
-pm_macSwitch("co2cement_process") =0 ;
-pm_macCostSwitch("co2cement_process") =0 ;
 
 *** load econometric emission data
 *** read in p3 and p4
-table p_emineg_econometric(all_regi,all_enty,p)        "parameters for ch4 and n2o emissions from waste baseline and co2 emissions from cement production"
+*** the co2cement_process part is only used in subsectors
+table pm_emineg_econometric(all_regi,all_enty,p)        "parameters for ch4 and n2o emissions from waste baseline and co2 emissions from cement production"
 $ondelim
 $include "./core/input/p_emineg_econometric.cs3r"
```
Comment (LaviniaBaumstark) [inline core/datainput.gms:1367]: this should be renamed in the input data generation but maybe we can keep it as is at it will be deleted anyway

**Comment by LaviniaBaumstark** [comment]
what is the status of this PR?

**Pull request closed without merging**